src/assets/styles/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$color-bg: #f8f9ff;
+$color-border: #dde2ff;
+$color-primary: #7a8dfd;
+$color-white: #ffffff;
```

Comment:
Move color variables to color specific scss file.

i.e _colors.scss

Then maybe we can import them here. _variables is too generic and usually we import other variables modules (like spacing) in _variables.scss